### PR TITLE
Fixes profile level headings to only increase by 1

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by tmitchellgcio** [comment]
Setting to WIP to check on tests and be sure the ones that should pass are passing

File: src/applications/personalization/profile/components/ProfileInfoTable.jsx [hunk 1]
```diff
@@ -82,7 +84,11 @@ const ProfileInfoTable = ({
   // can be passed directly to a `className` attribute
   const classes = {
     table: ['profile-info-table', className].join(' '),
-    title: [...titleClasses, ...titleClassesMedium].join(' '),
+    title: [
+      'vads-u-font-size--h3',
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/ProfileInfoTable.jsx:88]: Can you move this new class to the `titleClasses` array instead?

File: src/applications/personalization/profile/components/ProfileInfoTable.jsx [hunk 2]
```diff
@@ -17,6 +18,7 @@ const ProfileInfoTable = ({
   title,
   className,
   namedAnchor,
+  level = 3, // heading level
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/ProfileInfoTable.jsx:21]: Can you add this to the prop types? Extra credit for making sure it’s a number from 1-6

Comment (tmitchellgcio) [reply src/applications/personalization/profile/components/ProfileInfoTable.jsx:22]: done

File: src/applications/personalization/profile/components/ProfileInfoTable.jsx (hunk 1 above)
Comment (tmitchellgcio) [reply src/applications/personalization/profile/components/ProfileInfoTable.jsx]: yep sorry about that

File: src/applications/personalization/profile/components/ProfileInfoTable.jsx [hunk 3]
```diff
@@ -30,6 +31,7 @@ const ProfileInfoTable = ({
     'margin--0',
     'padding-x--2',
     'padding-y--1p5',
+    'vads-u-font-size--h3',
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/ProfileInfoTable.jsx:34]: Whoops, this no longer needs the `vads-u` prefix

File: src/applications/personalization/profile/components/ProfileInfoTable.jsx [hunk 4]
```diff
@@ -138,6 +146,7 @@ ProfileInfoTable.propTypes = {
   dataTransformer: PropTypes.func,
   className: PropTypes.string,
   namedAnchor: PropTypes.string,
+  level: numberBetween(1, 6),
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/ProfileInfoTable.jsx:149]: Oh that's nice and easy to read :)

File: src/applications/personalization/profile/components/ProfileInfoTable.jsx (hunk 3 above)
Comment (tmitchellgcio) [reply src/applications/personalization/profile/components/ProfileInfoTable.jsx:34]: no problem will adjust that

**Review by erikphansen — APPROVED** [review]
Looks solid!

**Comment by erikphansen** [comment]
@tmitchellgcio even all the optional checks passed so you get a juicy green button to press

**Pull request merged**